element_electrode_localization/coordinate_framework.py

```diff
@@ -0,0 +1,68 @@
+import datajoint as dj
+
+
+schema = dj.schema()
+
+
+def activate(schema_name, *, create_schema=True, create_tables=True):
+    """
+    activate(schema_name, create_schema=True, create_tables=True)
+        :param schema_name: schema name on the database server to activate the `coordinate_framework` element
+        :param create_schema: when True (default), create schema in the database if it does not yet exist.
+        :param create_tables: when True (default), create tables in the database if they do not yet exist.
+    """
+    schema.activate(schema_name, create_schema=create_schema, create_tables=create_tables)
+
+
+@schema
+class CCF(dj.Lookup):
+    definition = """  # Common Coordinate Framework
+    # CCF Dataset Information
+    ccf_id:             int             # CCF ID
+    ---
+    ccf_version:        int             # Allen CCF Version - e.g. CCFv3
+    ccf_description='': varchar(255)    # CCFLabel Description
+    """
+
+    class Voxel(dj.Part):
+        definition = """  # CCF voxel coordinates
+        -> master
+        x   :  int   # (um)  Anterior-to-Posterior (AP axis)
+        y   :  int   # (um)  Superior-to-Inferior (DV axis)
+        z   :  int   # (um)  Left-to-Right (ML axis)
+        index(y, z)
```

Comment:
Hi @ttngu207, what's the goal with the `index` option here?

Reply:
To improve queries. This table is massive, containing essentially the volume cloud of the mouse brain, at potentially 10um resolution. 
The default indexing is with `(x, y, z)`, so that search is fast, but if users search by y and z only, it would be considerably slower, adding this index helps improve that